src/components/Home.jsx

```diff
@@ -0,0 +1,36 @@
+import Radium from 'radium';
+import React from 'react';
+
+let Link = require('react-router-dom').Link;
+Link = Radium(Link);
+
+class Home extends React.Component {
+  render() {
+    return (
+      <ul style={styles.base}>
+        <li style={styles.li}>
+          <Link to="/create" style={styles.a}>Create an Event</Link>
```

Comment:
Semantics here maybe but perhaps path should be `/new`?

Reply:
Yeah, that sounds better.